tests/manual-test-cases/Group11-Stress/11-4-Many-Volumes.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Documentation  Test 11-4-Many-Volumes
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server
+Suite Teardown  Cleanup VIC Appliance On Test Server
+
+*** Test Cases ***
+Docker volume create 1000 volumes rapidly
+    ${pids}=  Create List
+
+    # Create 1000 volumes rapidly
+    :FOR  ${idx}  IN RANGE  0  1000
+    \   ${pid}=  Start Process  docker ${params} volume create --name\=multiple${idx} --opt Capacity\=1MB  shell=True
```

Comment:
This should not be 1MB because VSAN has major issues with sizes this small. Maybe 128MB?